docs/general/serializers.md

```diff
@@ -74,6 +74,55 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+As of release `0.10`, polymorphic relationships are serialized just by specifying the relationship. For example:
+
+```ruby
+class PictureSerializer < ActiveModel::Serializer
+  has_one :imageable
+end
+```
+
+If the instance of `Picture` belongs to a `Product`, an example payload might be:
+
+```javascript
+{
+  picture: {
+    id: 1,
+    imageable: {
+      id: 3,
+      title: "Product 3"
+    }
+  }
+}
+```
+
+In prior releases (and optionally in `0.10`), a `polymorphic: true` parameter is required:
```

Comment:
After investigation, there is no trace of any `polymorphic` option in 0.10-rc1+.